SpatialGDK/Source/SpatialGDK/Private/Interop/LoadBalancingWriter.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/LoadBalancingWriter.h"
+
+#include "EngineClasses/SpatialActorChannel.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "EngineClasses/SpatialPackageMapClient.h"
+#include "LoadBalancing/AbstractLBStrategy.h"
+#include "Utils/SpatialActorUtils.h"
+
+namespace SpatialGDK
+{
+void FLoadBalancingWriterBase::SendComponentUpdate(Worker_EntityId EntityId, FWorkerComponentUpdate& ComponentUpdate)
+{
+	NetDriver->Connection->SendComponentUpdate(EntityId, &ComponentUpdate);
+}
+
+class LoadBalancingWriterActorSet : public TLoadBalancingWriter<ActorSetMember>
+{
+public:
+	explicit LoadBalancingWriterActorSet(USpatialNetDriver* NetDriver)
```

Comment:
Avoid having anything use net driver directly. In this case I think it needs `ViewCoordinator`